Fix build with pango 1.50 and later:

In file included from pango-xsl-attributes.c:25:
./pango-xsl-attributes.h:116:18: error: conflicting types for 'pango_attr_line_height_new'
PangoAttribute * pango_attr_line_height_new (gint line_height);
                 ^
/usr/local/include/pango-1.0/pango/pango-attributes.h:607:25: note: previous declaration is here
PangoAttribute *        pango_attr_line_height_new              (double                       factor);
                        ^
pango-xsl-attributes.c:620:1: error: conflicting types for 'pango_attr_line_height_new'
pango_attr_line_height_new (gint line_height)
^
/usr/local/include/pango-1.0/pango/pango-attributes.h:607:25: note: previous declaration is here
PangoAttribute *        pango_attr_line_height_new              (double                       factor);
                        ^

https://cgit.freebsd.org/ports/commit/?id=20c0152349bce047bb3198451234946f849f3d6b
--- pangoxsl/pango-xsl-attributes.h.orig	2004-10-10 23:44:39 UTC
+++ pangoxsl/pango-xsl-attributes.h
@@ -25,6 +25,7 @@
 #define __PANGO_XSL_ATTRIBUTES_H__
 
 #include <pango/pango-attributes.h>
+#include <pango/pango-utils.h>
 
 G_BEGIN_DECLS
 
@@ -113,7 +114,9 @@ PangoAttrType    pango_attr_keep_with_previous_within_
 PangoAttribute * pango_attr_keep_with_previous_within_line_new (gint strength);
 
 PangoAttrType    pango_attr_line_height_get_type (void);
+#if !PANGO_VERSION_CHECK(1,50,0)
 PangoAttribute * pango_attr_line_height_new (gint line_height);
+#endif
 
 PangoAttrType    pango_attr_line_stacking_strategy_get_type (void);
 PangoAttribute * pango_attr_line_stacking_strategy_new (PangoLineStackingStrategy line_stacking_strategy);
--- pangoxsl/pango-xsl-attributes.c.orig	2004-10-10 23:44:39 UTC
+++ pangoxsl/pango-xsl-attributes.c
@@ -616,6 +616,7 @@ pango_attr_line_height_get_type (void)
  * 
  * Return value: the new #PangoAttribute.
  **/
+#if !PANGO_VERSION_CHECK(1,50,0)
 PangoAttribute *
 pango_attr_line_height_new (gint line_height)
 {
@@ -631,6 +632,7 @@ pango_attr_line_height_new (gint line_height)
 
   return pango_attr_int_new (&klass, line_height);
 }
+#endif
 
 /**
  * pango_attr_line_stacking_strategy_get_type:
